### ISSUE TITLE
repo moved?

### ISSUE THREAD

**Issue by larshp** [Daplie/greenlock-express#125]
Hi,
Has the maintenance for this project moved to https://git.coolaj86.com/coolaj86/greenlock-express.js ?

Consider updating the readme, and setting the repo as archived, https://help.github.com/articles/archiving-a-github-repository/

https://git.coolaj86.com says "Sign in with Github", however it does not let me sign in, instead it asks for creating a user and password.

**Comment by coolaj86** [comment]
Yes. I am the owner and maintainer and that is the correct repo.

As for the login confusion, yes, the UX is a little confusing and I've opened an issue about it: https://github.com/go-gitea/gitea/issues/3837

You simply need to add a recovery password to the account to finish creating it, so it looks a little more complicated than it really is.

**Comment by larshp** [comment]
Thanks, consider setting the repo as archived, and updating the readme, to avoid confusion

**Comment by coolaj86** [comment]
I'm not the owner of this fork.
